### DISCUSSION TITLE
22621.105_220617

### DISCUSSION THREAD

**Discussion by WhatTheBlock** [WhatTheBlock/WindowsSimplify#29] [Polls]
**Preview：**
![1](https://raw.githubusercontent.com/WhatTheBlock/WindowsSimplify/master/preview/22000.593_220404_1.png)
![2](https://raw.githubusercontent.com/WhatTheBlock/WindowsSimplify/master/preview/22621.105_220617.png)

#### Release Date：2022/06/17

**Edition：** Windows 11 Pro

**OS Build：** 22621.105

**ISO Size：** 2.53GB (Ver.1)、2GB (Ver.2)

**Download：** [Link](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w11.220617)

**SHA-256：**
- Ver.1: 3c927c9507a9058b300ea3261c94a8f8107fec0d67dc9bf96efb330349c58fab
- Ver.2: 03429180f77223416c25ccec6119f882d0b37d4b8d9a171a870966735f219b6a

**Highlights：**
- Rectify 11 theme (include windows setup)
- Very Stable (Ver.1)
- Preinstall VC++ Runtime
- Debloated

**Comment by xinyan1022** [comment]
請問大大可以釋出地址嗎?期待兩個月了 QQ

**Reply by WhatTheBlock** [reply]
移除WinSxS的ver.2已上傳
使用前請先在虛擬機測試是否符合需求

**Reply by xinyan1022** [reply]
大大 稍早之前下載完畢進行安裝!不過在安裝畫面到81%會顯示安裝錯誤 這個我不知道哪個環節出了問題!
所以先退回之前四月份的版本

**Reply by WhatTheBlock** [reply]
我用虛擬機測試是正常的，建議用虛擬機測一次看是不是檔案有問題

**Reply by xinyan1022** [reply]
剛剛用虛擬機測試!是可以正常安裝的!
一會兒我在實際安裝一次

**Reply by WhatTheBlock** [reply]
ver.1已上傳，建議使用此版本

**Reply by xinyan1022** [reply]
我重新下載V2安裝還是一樣出錯 哭啊
現在下載V1 等等反饋其況
謝謝版主

**Reply by xinyan1022** [reply]
不 這對強迫症的習慣來說是一場惡夢啊 兩個都不能安裝 嗚
![Uploading 73C77764-BE11-4F26-AF0D-BC99BC85DF9E.jpeg…]()

**Reply by xinyan1022** [reply]
手機沒辦法上傳圖片 真糟 虛擬機測試是正常的
無緣安裝這個版本！遺憾啊😩

**Reply by WhatTheBlock** [reply]
巴哈留言可以上傳圖片，再麻煩提供錯誤畫面讓我看看

**Reply by xinyan1022** [reply]
有上圖了 兩個版本安裝過程中都是卡在同一個地方

**Comment by ja12yang** [comment]
It's a great OS

**Comment by osunday** [comment]
Can gaming work on Ver 1 and is WinSxs included on the same version?

**Reply by WhatTheBlock** [reply]
Yes, Ver.1 is very stable and can be used for any needs.

**Reply by WhatTheBlock** [reply]
Ver.1 has been uploaded. It is recommended to test it on a virtual machine first.

**Reply by osunday** [reply]
What about hibernation?

**Reply by WhatTheBlock** [reply]
I forgot if it was disabled. You can use dism++ to choose whether to enable or not.

**Comment by osunday** [comment]
Please release 23H2 insider version

**Reply by WhatTheBlock** [reply]
I will, but I've been busy lately.

**Comment by lhga** [comment]
此精簡版，美觀穩定，還可以系統連綫更新。
唯一不足：遠端桌面無法開啓。可有修復補丁？
希望以後的版本保留遠端桌面功能。

**Reply by WhatTheBlock** [reply]
下次製作會保留此功能，謝謝
